service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/BehandlingStatistikkMapper.kt

```diff
@@ -283,21 +382,27 @@ class BehandlingStatistikkMapper(
 
         internal fun map(revurdering: Revurdering): ResultatOgBegrunnelse = when (revurdering) {
             is IverksattRevurdering.Innvilget -> ResultatOgBegrunnelse(innvilget, null)
-            is IverksattRevurdering.Opphørt -> ResultatOgBegrunnelse(opphørt, listUtOpphørsgrunner(revurdering.utledOpphørsgrunner()))
+            is IverksattRevurdering.Opphørt -> ResultatOgBegrunnelse(
+                opphørt,
+                listUtOpphørsgrunner(revurdering.utledOpphørsgrunner()),
+            )
             is IverksattRevurdering.IngenEndring -> ResultatOgBegrunnelse(ingenEndring, ingenEndringBegrunnelse)
             else -> throw ManglendeStatistikkMappingException(this, revurdering::class.java)
         }
 
-        fun map(stansAvYtelse: StansAvYtelseRevurdering.IverksattStansAvYtelse) = ResultatOgBegrunnelse(stans, stansAvYtelse.revurderingsårsak.årsak.hentGyldigStansBegrunnelse())
+        fun map(stansAvYtelse: StansAvYtelseRevurdering.IverksattStansAvYtelse) =
+            ResultatOgBegrunnelse(stans, stansAvYtelse.revurderingsårsak.årsak.hentGyldigStansBegrunnelse())
```

Comment:
Finnes det også ugyldige stans-begrunnelser? :grimacing:

Reply:
Ja, per nå så skal grunnen for stans kun være manglende kontrollerklæring og den enaste grunnen for å gjenoppta er att man har mottat kontrollerklæring osv. 😊